doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -0,0 +1,136 @@
+.. _deploy-cephadm-mgmt-gateway:
+
+=============
+Mgmt Gateway
+=============
+
+Deploying mgmt-gateway
+======================
+
+Starting from version 19.x.x, the `mgmt-gateway` service introduces a new design for Ceph applications
+based on a modular, service-based architecture. This service, managed by cephadm and built on top of nginx
+(an open-source, high-performance web server), acts as the new front-end and single entry point to the
+Ceph cluster. The `mgmt-gateway` provides unified access to all Ceph applications, including the ceph dashboard
+and monitoring stack. Utilizing nginx enhances security and simplifies access management due to its robust
+community support and high-security standards. The `mgmt-gateway` service acts as a reverse-proxy that routes
+requests to the appropriate ceph backend application.
+
+In order to deploy the mgmt-gateway service, use the following command:
+
+.. prompt:: bash #
+
+    ceph orch apply mgmt-gateway [--placement ...] ...
+
+Once applied cephadm will automatically reconfigure several running daemons (including monitoring)
+to run automatically behind the new created service. External access to those services will not be possible
+anymore. All the access will be centralized through the new created service endpoint: `https://<node-ip>:<port>`.
+
+
+Benefits of the mgmt-gateway service
+====================================
+* ``Unified Access``: Centralizing access through nginx improves security and provide a single entry to the cluster.
+* ``Improved user experience``: User shouldn't care anymore about where each application is running (ip/host).
+* ``High Availability for dashboard``: nginx HA mechanisms are used to provide high availability for ceph dashboard.
+* ``High Availability for monitoring``: nginx HA mechanisms are used to provide high availability for monitoring.
+
+Security enhancements
+=====================
+
+Once the `mgmt-gateway` service is deployed user cannot access monitoring services without authentication through the
+dashboard.
+
+
+High availability enhancements
+==============================
+nginx HA mechanisms are used to provide high availability for all the Ceph management applications including Dashboard
+and monitoring stack. In case of the Dashboard user shouldn't care anymore where the active manager is running.
+`mgmt-gateway` handles manager failover transparently and redirects the user to the active manager. In case of the
+monitoring `mgmt-gateway` takes care of handling the high availability in case several instances of Prometheus, Alertmanager
+or Grafana are available. The reverse-proxy will automatically detect healthy instances and use them to process user requests.
+
+
+Accessing services with mgmt-gateway
+====================================
+
+Once the `mgmt-gateway` service is deployed direct access to the monitoring services will not be allowed anymore.
```

Comment:
will no longer be possible

(but why?)

Reply:
@anthonyeleven because mTLS will will be enabled and it will prevent any external access to those applications.